### ISSUE TITLE
Dymo Labelwriter wireless

### ISSUE THREAD

**Issue by meosborne** [michaelrsweet/lprint#23]
While a driver is listed for the Dymo Labelwriter wireless (LWW), I can't get anything to print on the one I have. I don't get any errors, but nothing prints via usb or network. Looking at the code, it looks like the same driver is being used as for the LW 450 and others. CUPS can't print to the LWW either. 

The only thing I've found that can print to the LWW is this project: https://github.com/minlux/dymon. Based on the info there it seems that the LWW is using a different protocol than earlier LWs. :(

Unfortunately, Dymo has not issued any guides for this model that I am aware of. The above project has reverse engineered some of the protocol though.

Is the LWW supposed to work?

**Comment by michaelrsweet** [comment]
Hmm, I'll need to investigate further, as I don't have this particular printer...

**Comment by meosborne** [comment]
Oops, forgot to mention that I'm using Linux, Fedora 32. Also, many thanks for all your work on this, CUPS, ESP Print Pro, and everything else!

**Comment by michaelrsweet** [comment]
@meosborne OK, so I found more information here: <https://developers.dymo.com>

The wireless and 5xx series printers use an updated command set. For now I am going to comment out the LabelWriter Wireless driver, but it looks like it should be fairly straight-forward to add it if you are available to test?

**Comment by meosborne** [comment]
I would be happy to test.

**Comment by bartlibert** [comment]
I can test as well

**Comment by Gandalf-the-Blue** [comment]
I'm in the market for a Dymo LabelWrite 550 and was wondering if this issue is resolved/being worked upon currently?

**Comment by michaelrsweet** [comment]
@Gandalf-the-Blue I haven't started working on this driver yet so I can't say when it will be completed...

**Comment by kemes** [comment]
@michaelrsweet How does the development of the LW 550 driver look atm? I have a 550 sitting on my desk and can provide help with testing if needed.

**Comment by milkman0007** [comment]
What's going up be new on 1.3 ?

**Comment by michaelrsweet** [comment]
@milkman0007 From the CHANGES.md file:

- Added new dithering algorithm to better support barcode printing with shaded content.
- Added experimental Brother printer support (Issue #15)
- Added basic TSPL printer support (Issue #54)
- Added basic SEIKO printer support (Issue #58)
- Added experimental Zebra PCL printer support.
- Added support for configuration files in "/etc", "/usr/local/etc", or "/Library/Application Support" (macOS).
- Updated ZPL driver to report 'media-needed' reason when out of labels during a job.
- Fixed copies support for ZPL printers (Issue #100)
- Fixed darkness calculations for EPL and ZPL printers (Issue #104)

**Comment by milkman0007** [comment]
Nice any you tell more about the pcl support ?

**Comment by michaelrsweet** [comment]
@milkman0007 I ported the CPCL (sorry typo) driver from the old CUPS rastertolabel driver. I was also going to port the Intellitech PCL driver but they have gone out of business... :/

**Comment by milkman0007** [comment]
Yeah I just saw intellitech close the door anyone know what happened ? 
Didn't they manage some label apps too ?
